package.json

```diff
@@ -11,6 +11,7 @@
     "test": "craco test --env=jsdom",
     "start:ssl": "HTTPS=true yarn start",
     "start:service-worker": "yarn build && yarn serve",
+    "lint:fix": "yarn run eslint --fix ./src",
```

Comment:
i thought we had it, but should we do one to do the lint without the fix?

Reply:
the changes affect files outside of the `src`

i guess you did it on porpouse to apply the rules to things outside, but leave the script to fix only src?

Reply:
In CI we use `wearerequired/lint-action@v2` to run eslint.
Actually, I just forgot to make `./` instead of `./src`. Thanks!